x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_flyout.test.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import '../../../__mocks__/kea.mock';
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { useValues, useActions } from 'kea';
+
+import { CredentialsFlyout } from './credentials_flyout';
+import { EuiFlyout } from '@elastic/eui';
+import { IApiToken } from './types';
+import { PRIVATE } from './constants';
+
+describe('Credentials', () => {
+  const apiToken: IApiToken = {
+    name: '',
+    type: PRIVATE,
+    read: true,
+    write: true,
+    access_all_engines: true,
+  };
+
+  const mockKea = ({ values = {}, actions = {} }) => {
+    const mergedValues = {
+      activeApiToken: apiToken,
+      activeApiTokenIsExisting: false,
```

Comment:
This should be `activeApiTokenExists` now right?

Reply:
It should. Looks like it's not being used anywhere, which is why it is still allowed.

Would be a good case for strong typing stuff like this in tests, so that we get dinged if we forget to update tests after a name change.